src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/ShellInProcess.cs

```diff
@@ -142,6 +143,13 @@ public async Task WaitForFileChangeNotificationsAsync(CancellationToken cancella
             }
         }
 
+        public async Task ExecuteCommandAsync(string commandName, string argument, CancellationToken cancellationToken)
+        {
+            await JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
+            var dte = await GetRequiredGlobalServiceAsync<SDTE, EnvDTE.DTE>(cancellationToken);
+            dte.ExecuteCommand(commandName, argument);
```

Comment:
@sharwell wanted to check with you if this was appropriate - this is how the old framework did it, but the new framework uses a different method - https://github.com/microsoft/vs-extension-testing/blob/d460a813832a479f21ec0d076de213e6a1a5ccae/src/Microsoft.VisualStudio.Extensibility.Testing.SourceGenerator.UnitTests/Resources/TestGenerationForVS2022/ShellInProcess1.g.cs#L61

However I didn't see (or understand perhaps) a way to pass args in the way that the new framework is doing it.

Reply:
The new approach intentionally avoids DTE. We can update it to support arguments tomorrow. 👍